webapp/src/components/conference/index.ts

```diff
@@ -0,0 +1,26 @@
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+
+import {GlobalState} from 'mattermost-redux/types/store';
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import {id as pluginId} from '../../manifest';
+import Conference from './conference';
+import {openJitsiMeeting} from '../../actions';
+
+function mapStateToProps(state: GlobalState) {
+    return {
+        post: (state as any)[`plugins-${pluginId}`].openMeeting,
```

Comment:
Is the use of `any` the only reasonable way to access this?

Reply:
I guess I can extend the global state, let me try.